raster-foundry/raster-foundry
# Allow specifying mosaic color composites via url params

## Overview

This PR adds url parameter handling to the mosaic tile route that allows users to specify color composites via `redBand`, `greenBand`, and `blueBand` paramters (all ints).

All three params must be specified for it to take effect, otherwise it falls back to the composites defined by the scenes-to-projects.

### Checklist



## Testing Instructions

 * Get a request for a tile
 * Using curl or httpie or something, add the `redBand`, `greenBand`, `blueBand` parameters to the URL and make sure it works as expected.

Closes #3845
